content/agent/faq/how-do-i-uninstall-the-agent.md

```diff
@@ -0,0 +1,114 @@
+---
+title: How do I uninstall the agent?
+kind: faq
+further_reading:
+- link: "/agent/"
+  tag: "Documentation"
+  text: Learn more about the Datadog Agent
+aliases:
+  - /agent/faq/how-do-i-uninstall-the-agent/
+---
+
+## Agent v6
+### Amazon Linux/Debian/Fedora/Ubuntu
+```
+$ sudo apt-get --purge remove datadog-agent -y
+```
+
+### CentOS
+
+* For CentOS 5:
+
+    ```
+    $ sudo yum remove datadog-agent-base
+    ```
+
+* For CentOS 6:
+
+    ```
+    $ sudo yum remove datadog-agent
+    ```
+
+### Mac OS
+
+Stop and Close the Datadog Agent: via the bone icon in the Tray.
+
+Drag the Datadog Application from the application folder to the Trash Bin.
+
+```
+$ sudo rm -rf /opt/datadog-agent
+$ sudo rm -rf /usr/local/bin/datadog-agent
+$ sudo rm -rf ~/.datadog-agent/**​ #to remove broken symlinks
+```
+If you ran the optional install commands to have the Agent run at boot time, run the following to finish uninstalling:
+
+```
+$ sudo launchctl unload -w /Library/LaunchDaemons/com.datadoghq.agent.plist
+$ sudo  rm /Library/LaunchDaemons/com.datadoghq.agent.plist
+```
+
+### Redhat
+
+```
+$ sudo yum remove datadog-agent
+```
+
+### Windows
+
+For Windows, it's important that the original account used to install the agent is also used to remove it, otherwise it’s possible remnants will be left behind and it will not be cleanly removed.
+
+Uninstall the agent using Add/Remove Programs, alternatively, it's possible to to use Powershell as well. Here is a one liner:
```

Comment:
Where are the "Add/Remove Programs" located?

I'm not a Windows user, but looking at this from a high-level I think it would benefit from a bit more explanation.

Reply:
for a windows user is pretty trivial actually :)

Reply:
Ahh, gotcha